tests/utils.js

```diff
@@ -881,7 +881,8 @@ module.exports = {
    * @returns {Object} that contains the promise to resolve when logs lines are matched and a cancel function
    */
   waitForLogs: (logFilename, ...regex) => {
-    const tail = new Tail(`./tests/logs/${logFilename}`);
+    const name = constants.IS_TRAVIS ? 'horti.log' : logFilename;
```

Comment:
Can we keep the logs separate? It's much easier for debugging.

Reply:
Although, is some value (for debugging) in having logs in the same file, being able to see what the other service is using while another errors out. I think we should make it very evident which logs are coming from which service! 

My absolutely preferred way would be to keep logs separate and always have timestamps. 
No need to do that now, it's just my personal preference.